packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts

```diff
@@ -1,26 +1,33 @@
-import { sprinkles, vars } from '@kadena/react-ui/theme';
+import { atoms, darkThemeClass, tokens } from '@kadena/react-ui/theme';
 import { style } from '@vanilla-extract/css';
 import { $$footerMenu, $$pageWidth } from '../../global.css';
 
 export const footerWrapperClass = style([
-  sprinkles({
+  atoms({
     position: 'relative',
-    backgroundColor: '$gray90',
+    backgroundColor: 'layer-1.inverse.default',
   }),
   {
-    marginBlockStart: vars.sizes.$40,
+    selectors: {
+      [`${darkThemeClass} &`]: {
+        backgroundColor:
+          tokens.kda.foundation.color.background['layer-1'].default,
+      },
+    },
```

Comment:
you should not need this becuase it should be applied correctly with the atoms utility class!